rust/operator-binary/src/controller.rs

```diff
@@ -1100,9 +1079,8 @@ fn build_metastore_rolegroup_statefulset(
                 ),
                 ..LabelSelector::default()
             },
-            service_name: Some(build_headless_role_group_metrics_service_name(
-                rolegroup_ref.object_name(),
-            )),
+            // TODO: Use method on RoleGroupRef once op-rs is released
+            service_name: Some(rolegroup_metrics_service_name(rolegroup_ref)),
```

Comment:
Should be the headless service.
```suggestion
            service_name: Some(rolegroup_headless_service_name(rolegroup_ref)),
```